getting_started/setup_your_environment.md

```diff
@@ -106,88 +51,59 @@ more information on how to set-up your JetBrains IDE for Deno, read
 [this comment](https://youtrack.jetbrains.com/issue/WEB-41607#focus=streamItem-27-4160152.0-0)
 on YouTrack.
 
-#### Vim and NeoVim
+JetBrain's is considering migrating to the Deno language server (see:
+[YouTrack WEB-48625](https://youtrack.jetbrains.com/issue/WEB-48625)). If you
+are a user of JetBrains and Deno, voicing your support can continue help
+JetBrains prioritize support.
+
+#### vim/Neovim
+
+Deno is well supported on both [vim](https://www.vim.org/) and
+[Neovim](https://neovim.io/) via
+[coc.nvim](https://github.com/neoclide/coc.nvim) and
+[ALE](https://github.com/dense-analysis/ale). coc.nvim offers plugins to
+integrate to the Deno language server while ALE supports it _out of the box_.
 
```

Comment:
The official config for Neovim 0.5 is provided by https://github.com/neovim/nvim-lspconfig and is detailed at https://github.com/neovim/nvim-lspconfig/blob/master/CONFIG.md#denols

Reply:
I added this in #57